src/lib/parser_utils.rs

```diff
@@ -10,12 +10,19 @@ pub fn get_one<T>(d: Delim<T>) -> Result<Box<T>, Delim<T>> {
     }
 }
 
-pub fn dec_into_exp(d: Dec) -> Exp {
-    match d {
-        Dec::Exp(e) => e,
-        d => Exp::Block(Delim {
-            vec: vec![d],
-            has_trailing: false,
+pub fn dec_into_exp(d: Dec_) -> Exp_ {
+    Loc(
```

Comment:
Same question, and more evidence for combining I think.

Reply:
Here's a [counterexample](https://github.com/dfinity/motoko.rs/pull/16/files#diff-281a47c6058d9c328679c9a909440673cf1459a8987782812214ac883fb19791R88-R115) where it makes sense to use `Loc<_>` without `Box<_>`. We could also eventually use this to optimize the VM, since boxing would add a bit of extra overhead which could make an impact on the interpreter's performance. Seems worth having a "separation of concerns" similar to Rust's `Arc<Mutex<_>>` pattern for reusability in different situations. I'll definitely add a shorthand for `Loc<Box<_>>` though, since it will probably continue to be used in most cases.